Makefile

```diff
@@ -12,7 +12,7 @@ NAME := netplugin
 # We are using date based versioning, so for consistent version during a build
 # we evaluate and set the value of version once in a file and use it in 'tar'
 # and 'release' targets.
-VERSION_FILE := /tmp/$(NAME)-version
+VERSION_FILE := $(NAME)-version
```

Comment:
This was needed because this file was only on the VM, not on the host's file system.